frontend/catalyst/pennylane_extensions.py

```diff
@@ -110,20 +111,38 @@ class QFunc:
     """
 
     def __init__(self, fn, device):
-        self.func = fn
-        self.device = device
-        update_wrapper(self, fn)
+        self.func = fn  # pragma: nocover
+        self.device = device  # pragma: nocover
+        update_wrapper(self, fn)  # pragma: nocover
```

Comment:
Nit:
```suggestion
    def __init__(self, fn, device):  # pragma: nocover
        self.func = fn	        self.func = fn  
        self.device = device	        self.device = device
        update_wrapper(self, fn)	        update_wrapper(self, fn)
```

Reply:
Can you double check this?

Reply:
Are you referring to the duplicated lines? 😅 I think that's just what happens when you force a ` ```suggestion` across a changed and unchanged code block. I just wanted to indicate that the `nocover` can be moved to the block, rather than marking each individual statement within.

Reply:
No, I am talking about:

```python
self.func = fun                   self.func = fn
```

I'll just delete the one on the right.

Reply:
I edited your suggestion.

Reply:
I'm marking this as resolved. Did change it, but don't remember the commit.